torch/nn/modules/module.py

```diff
@@ -1197,7 +1200,9 @@ def load(module, prefix=''):
                     load(child, prefix + name + '.')
 
         load(self)
-        load = None  # break load->load reference cycle
+        # instruction "load = None" => break load->load reference cycle
+        # type ignore[...] is required because mypy isn't happy with the redef of "load"
+        load = None  # type: ignore[assignment]
```

Comment:
I am not python expert. but cant we use `del load` to remove the function reference itself? (after all the load has been done, yes?)

Reply:
Yes!